docs/httpproxy.md

```diff
@@ -750,7 +750,172 @@ spec:
 
 ## HTTPProxy inclusion
 
-TODO(youngnick) document inclusion
+HTTPProxy permits the splitting of a system's configuration into separate HTTPProxy instances using **inclusion**.
+
+Inclusion, as the name implies, allows for one HTTPProxy object to be included in another, optionally with some conditions around the inclusion.
+Contour reads the inclusion tree and merges the included objects into one big object internally before rendering Envoy config.
+Importantly, the included HTTPProxy objects do not have to be in the same namespace, so this is functionally the same as the old delegation feature of the now-deprecated IngressRoute.
+
+The "root" HTTPProxy is the only entry point for a proxied virtual host and is used as the top level configuration of a cluster's proxy resources.
```

Comment:
Maybe

Each tree of HTTPProxy starts with a root, the top level object of the configuration for a particular virtual host.